src/main/java/gtPlusPlus/xmod/gregtech/api/metatileentity/implementations/MTEHatchElementalDataOrbHolder.java

```diff
@@ -126,15 +126,15 @@ public void onScrewdriverRightClick(ForgeDirection side, EntityPlayer aPlayer, f
     public boolean allowPullStack(IGregTechTileEntity aBaseMetaTileEntity, int aIndex, ForgeDirection side,
         ItemStack aStack) {
         Logger.INFO("Checking if we can pull " + aStack.getDisplayName() + " from slot " + aIndex);
-        return aIndex == mInventory.length - 1 && ItemUtils.isControlCircuit(aStack)
+        return aIndex == mInventory.length - 1 && aStack.getItem() instanceof ItemIntegratedCircuit
```

Comment:
Sorry if I forgot to mention, I think these do also still need null checks
```suggestion
        return aIndex == mInventory.length - 1 && aStack != null && aStack.getItem() instanceof ItemIntegratedCircuit
```